AFQ/_fixes.py

```diff
@@ -228,6 +227,8 @@ def solve_qp(P, Q, G, H):
     x : array
         Optimal solution to the QP problem.
     """
+    import cvxpy as cvx
```

Comment:
Do you want to catch import errors here to provide a more helpful error message as you did for the different visualization backends? e.g.
```
try:
   import cvxpy as cvx
except ImportError:
   raise ImportError("Using MSMT in pyAFQ requires the the cvxpy package. You can install it using `pip install cvxpy` or `pip install pyAFQ[msmt]`.")
```
or some such.